great-idea/css/index.css

```diff
@@ -63,52 +63,84 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-nav {
-	display: inline-block;
+/*flex box*/
+.container {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
 }
 
-a {
-	color: lightgrey;
-	padding: 40px;
-	margin-right: 30px;
-	text-decoration: none;
+/*header*/
+header {
```

Comment:
Good job applying inheritance with the `header` and `cta` classes! The idea of inheritance and parent/child elements is really powerful, and it's something we'll keep coming back to in the course... so keep thinking about it and playing around with it.